middleware & hwameistor (#155)
DaoCloud/DaoCloud-docs: docs/en/middleware/elastic-search/intro/what.md

# What is Elasticsearch

> Elasticsearch (hereafter referred to as Elastic) , the current full-text search engine of choice, can quickly store, search and analyze a sea of data. Elastic is built on top of Lucene that cannot be used directly. It encapsulates Lucene and provids out-of-box REST APIs.

Reviewer: ```suggestion
Elasticsearch, the current full-text search engine of best choice, can quickly store, search, and analyze a set of data. Elasticsearch is built on top of Lucene that cannot be used directly. It encapsulates Lucene and provides out-of-box REST APIs.
```
英文不能缩写成 Elastic，人家不认识了，开会的时候有人简称 ES。不过书面用全称吧
